Import data backups to replace Submissions.

GoGoCarl opened the pull request:
Added a feature to import data backup ZIP files back
into to the data set in the submisssions folder. Two
options; one for appending data (great for aggregation)
and a second for overwriting (good for a first install
or a clean slate).

Translations needed to complete.

GoGoCarl committed:
Import data backups to replace Submissions.

Added a feature to import data backup ZIP files back
into to the data set in the submisssions folder. Two
options; one for appending data (great for aggregation)
and a second for overwriting (good for a first install
or a clean slate).

Translations needed to complete.

GoGoCarl commented:
Hopefully this merges cleanly -- this is the data import feature Conrad's been asking about.  Forgot that I had not merged it because I was waiting on translations (but didn't get around to asking for them!)  Once I get those back, I'll add them to this PR.  For now, posted this up for any preliminary testing, etc.

robheittman commented:
Doesn't merge cleanly, but I think I want to hold this one anyway until we've checked it thoroughly.

GoGoCarl committed:
Rebuild Monitoring.geojson on data import.

After a successful data import, rebuild the Monitoring.geojson
file based on the new contents of the file system.

The GeoJson builder itself can be called via:

    var Builder = require('./helpers/rebuild-geojson')
    Builder.generate(callback)

GoGoCarl commented:
@rfc2616 Agreed.  I've added the GeoJson rebuild logic here as well.

GoGoCarl committed:
Data Import tool translations for Spanish
